modules/@ergonode/ui/src/models/treeAccordion/index.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+
+export const getSelectedItems = value => value.reduce((prev, curr) => ({
+    ...prev,
+    [curr.id]: true,
+}), {});
+
+export const getSelectedNodesCount = ({
+    value,
+    treeStructure,
+}) => {
+    const nodesCount = {};
+
+    const count = (tree) => {
+        let sum = 0;
+
+        (tree.children || []).forEach((node) => {
+            sum += count(node);
+        });
+
+        const selectedValue = value.find(({
+            id,
+        }) => id === tree.id);
```

Comment:
if the order does not matter, move the variable declarations to the top of the function.

Reply:
it's reccurency, it's matter